Libs/Widgets/ctkSettingsDialog.cpp

```diff
@@ -167,11 +167,11 @@ void ctkSettingsDialogPrivate::updateRestartRequiredLabel()
   bool restartRequired = !restartRequiredSettings.isEmpty();
   if (restartRequired)
     {
-    QString header = q->tr(
-      "<b style=\"color:red\">Restart required!</b><br>\n<small>"
-      "The application must be restarted to take into account "
-      "the new values of the following properties:\n");
-    QString footer = q->tr("</small>");
+    QString header = "<b style=\"color:red\">" + ctkSettingsDialog::tr("Restart required!") + "</b><br>\n<small>";
+    header += ctkSettingsDialog::tr(
+      "The application must be restarted to take into account the new values of the following properties:\n"
```

Comment:
Could you move out the `\n` from the translatable string?

Reply:
Ok, it's done now, thanks !